data/mods/Magiclysm/enchanted/enchanted_belts.json

```diff
@@ -0,0 +1,186 @@
+[
+  {
+    "abstract": "mbelt_leather",
+    "type": "TOOL_ARMOR",
+    "name": "magic leather belt",
+    "description": "A leather belt.  Useful for making your pair of pants fit.",
+    "weight": "102 g",
+    "volume": "250 ml",
+    "price": 90000,
+    "bashing": 5,
+    "material": [ "leather" ],
+    "symbol": "[",
+    "color": "brown",
+    "covers": [ "TORSO" ],
+    "coverage": 5,
+    "material_thickness": 2,
+    "use_action": {
+      "type": "holster",
+      "holster_prompt": "Stick what into your belt",
+      "holster_msg": "You tuck your %s into your %s",
+      "max_volume": "500 ml",
+      "max_weight": 400,
+      "draw_cost": 60,
+      "flags": [ "BELT_CLIP" ]
+    },
+    "flags": [ "WAIST", "WATER_FRIENDLY", "STURDY" ]
+  },
+  {
+    "type": "TOOL_ARMOR",
+    "copy-from": "mbelt_leather",
+    "id": "mbelt_haste",
+    "name": "Belt of Haste",
+    "relic_data": {
+      "passive_effects": [ { "has": "WORN", "condition": "ALWAYS", "values": [ { "value": "SPEED", "multiply": 0.15, "add": 5 } ] } ]
+    }
+  },
+  {
+    "type": "TOOL_ARMOR",
+    "copy-from": "mbelt_leather",
+    "id": "mbelt_thor",
+    "name": "Megingjörð",
+    "description": "The mythical belt of Thor, god of thunder.  Or at least so it appears.  It doubles the wearer's base strength.",
+    "material": [ "superalloy" ],
+    "material_thickness": 10,
+    "environmental": 20,
+    "weight_capacity_bonus": "50 kg",
+    "relic_data": {
+      "passive_effects": [ { "has": "WORN", "condition": "ALWAYS", "values": [ { "value": "STRENGTH", "multiply": 1.0 } ] } ]
+    }
+  },
+  {
+    "type": "TOOL_ARMOR",
+    "copy-from": "mbelt_leather",
+    "id": "mbelt_pockets",
+    "name": "Girdle of Pockets",
+    "description": "A wide girdle that fits around your waist, coverd in numerous small pouches that hold a lot more than they should, and the weight of their contents is greatly reduced.",
+    "coverage": 10,
+    "encumbrance": 5,
+    "weight_capacity_bonus": "10 kg",
+    "storage": "15 L"
+  },
+  {
+    "type": "TOOL_ARMOR",
+    "copy-from": "mbelt_leather",
+    "id": "mbelt_weaponry",
+    "name": "Belt of Weaponry",
+    "description": "A wide girdle that fits around your waist, you can sheath or holster any weapon into it in the blink of an eye, and it seemingly stores them somewhere else.",
+    "coverage": 10,
+    "encumbrance": 3,
+    "weight_capacity_bonus": "10 kg",
```

Comment:
This will need to be revisited when the nested container system happens, as I am not quite happy with this implementation.